apps/nextjs-website/src/lib/fetch/types/StrapiImage.ts

```diff
@@ -0,0 +1,15 @@
+import * as t from 'io-ts';
+
+export const StrapiImageSchema = t.type({
+  data: t.union([
+    t.type({
+      attributes: t.type({
+        alternativeText: t.union([t.string, t.null]),
+        url: t.string,
+      }),
+    }),
+    t.null,
+  ]),
+});
```

Comment:
Use strict to have the output that contains only the defined attributes

```suggestion
export const StrapiImageSchema = t.strict({
  data: t.union([
    t.strict({
      attributes: t.strict({
        alternativeText: t.union([t.string, t.null]),
        url: t.string,
      }),
    }),
    t.null,
  ]),
});
```